proposals/distribution.md

```diff
@@ -0,0 +1,156 @@
+* Liu Genping <[liugenping@huawei.com](mailto:liugenping@huawei.com)>
+* Vanessa Sochat (@vsoch) (Stanford) <vsochat@stanford.edu>
+* Eduardo Arango (@ArangoGutierrez) (Sylabs) <eduardo@sylabs.io>
+
+## Code of Conduct
+
+This project would incorporate (by reference) the OCI Code of Conduct ([https://github.com/opencontainers/tob/blob/master/code-of-conduct.md](https://github.com/opencontainers/tob/blob/master/code-of-conduct.md)).
+
+## Governance and Releases
+
+This project would incorporate the Governance and Releases processes from the OCI project template: [https://github.com/opencontainers/project-template](https://github.com/opencontainers/project-template).
+
+## Project Communications
+
+Both of the proposed projects would continue to use existing channels in use by the OCI developer community for communication including:
+
+* GitHub for issues and pull requests
+* The dev@opencontainers.org email list
+* The monthly OCI developer community conference call
+* The #OpenContainers freenode IRC channel
+
+## Versioning / Roadmap
+
+The API spec is currently considered v2 and we will start the specification at v2.0. Fewer places to change and compare, and it would keep with it being a lateral move.
+
+## Frequently Asked Questions (FAQ)
+
+**Q: Does this include the code of the docker-registry?**
+
+A: No. This is an API specification discussion.
+
+**Q: Does this change the OCI Charter or Scope Table?**
+
+A: Not the charter, but it does change the scope table.
+This project is scoped to specifying per-image client ↔ registry interaction with an [HTTP][rfc7230]-based protocol.
+The following scope entries should be removed from the [scope table][scope]:
+
+* “Use of Hash as Content Addressable name for immutable containers”.
+    This entry is in scope for this project, and a more detailed entry will be added as described below.
+* “Creating Reference spec for optional DNS based naming & distribution”.
+    This entry conflates naming and distribution, which will be separated by this proposal.
+* “Standardizing on a particular Distribution method”.
+    This proposal will provide one (of possibly many) distribution specifications, so the old “There is no current agreement on how to distribute content” no longer applies.
+
+The following entries should be added to the [scope table][scope]:
+
+* “Specifying authentication and authorization schemes”.
+    Docker's current registry uses an [extension][token] of the [`Bearer`][rfc6750] [auth scheme][rfc7235-s2.1].
+    Work on specifying Docker's scheme will continue independently, and is orthogonal to the registry API.
+
+    * What: Specifying authentication and authorization schemes
+    * In/Out/Future: Out of scope
+    * Status: N/A
+    * Description: Defining protocols for authenticating and authorizing distribution access.
+    * Why: As an HTTP-based protocol, clients and servers can negotiate authentication via HTTP's [challenge-response authentication framework][rfc7235-s2.1].
```

Comment:
This should be specified in such a way that the bearer can be opaque to the registry implementation.

Reply:
> This should be specified in such a way that the bearer can be opaque to the registry implementation.

Can you file a PR against this branch or suggest alternative wording?  I think making negotiated authentication out of scope (as I've tried to do with this scope entry) is even stronger than “opaque to the registry implementation”.  More on this specific issue in #37, where I initially tried to make the `Bearer` auth part of the scope, but pivoted to the wording here based on @dmcgowan's [comment][1].

[1]: https://github.com/opencontainers/tob/pull/37#issuecomment-360923589

Reply:
> This should be specified in such a way that the bearer can be opaque to the registry implementation.

The mentioned rfc is broad enough to cover this case. This section is just saying the registry can return a response requesting a specific authentication type (could be bearer, basic, or whatever). The comment below this could probably be more generalized or removed completely as the rfc already describes this behavior in detail.